melos.yaml

```diff
@@ -46,7 +46,7 @@ scripts:
 
   test:
     description: Run tests in a specific package.
-    run: dart test -p vm,chrome
+    run: flutter test
```

Comment:
Will this still test `chrome`?

Reply:
I don't see chrome tests here https://github.com/powersync-ja/powersync.dart/actions/runs/9944203557/job/27469732966

Reply:
@stevensJourney Yeah the tests fail when its run with `dart -p chrome`. I will look into it again as the native side needs `flutter test` for loading the native extension files.